### PR TITLE
make approval/approver terms consistent

### PR DESCRIPTION
**Pull request by afeld**
Builds on #257.

https://github.com/18f/c2/compare/89169198_split_approvals-B...approval-simplify

### CODE REVIEW

File: app/models/cart.rb [hunk 1]
```diff
@@ -16,27 +16,19 @@ class Cart < ActiveRecord::Base
...
-  end
-
-  def approver_approvals
-    self.approvals.approvable
-  end
-
-  def approvers
-    self.approval_users.merge(self.approver_approvals)
+    self.approvals.rejected
   end
```
Comment (afeld) [inline app/models/cart.rb]: This is the main bit.

File: app/models/concerns/proposal_delegate.rb [hunk 2]
```diff
@@ -4,7 +4,10 @@ module ProposalDelegate
   included do
     belongs_to :proposal
     has_many :approvals, through: :proposal
-    has_many :approval_users, through: :approvals, source: :user
+    has_many :approvers, through: :approvals, source: :user
```
Comment (afeld) [inline app/models/concerns/proposal_delegate.rb]: Also this.

File: app/models/proposal.rb [hunk 3]
```diff
@@ -5,8 +5,10 @@ class Proposal < ActiveRecord::Base
 
   has_one :cart
   has_many :approvals
-  has_many :approval_users, through: :approvals, source: :user
+  has_many :approvers, through: :approvals, source: :user
```
Comment (afeld) [inline app/models/proposal.rb]: Also this.

**Comment by cmc333333** [comment]
Seems like a straight forward improvement

**Pull request merged**